src/action/github.ts

```diff
@@ -32,4 +34,14 @@ export class Github implements Action {
             }
         };
     }
+
+    getBaseBranchSha1(): string | null {
```

Comment:
I returned `string | null` from most endpoints, but perhaps a `Maybe` type would work better

Reply:
If for whatever reason the context is not set correctly then it could return undefined. It depends if an error is thrown during casting before it gets to that point.

Alternatively you could avoid null and just return undefined (again depends on how the casting library works if null would ever be allowed). Maybe<string> works for me too if you'd rather that approach.

Reply:
Should be a concern of the upstream library, no?

Reply:
It should be. I've double checked the types and it looks good.